samples/v1.5/Elements/SecondaryActions.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
looks like this sample file has a mix of spaces and tabs for indentation... our repo has a [`.editorconfig`](https://github.com/microsoft/AdaptiveCards/blob/0ceb34c39647f0fe811e51554466a7d417855a27/.editorconfig#L10-L14) that specifies tabs only for .json files...

(optional) you can find instructions for setting up your editor to consume the `.editorconfig` [here](https://editorconfig.org/#download)

Reply:
changes done.